docs/4.3/quickstart.md

```diff
@@ -274,6 +279,91 @@ $ echo "Awesome!"
 # check out your shared ssh session between two CLI windows
 ```
 
+## Run Teleport using Docker
+
+We provide pre-built Docker images for every version of Teleport. These images are hosted on quay.io.
+
+- [All tags under `quay.io/gravitational/teleport` are Teleport Community images](https://quay.io/repository/gravitational/teleport?tag=latest&tab=tags)
+
+We currently only offer Docker images for `x86_64` architectures.
+
+### Pick your image
+
+This table gives an idea of how our image naming scheme works. We offer images which point to a static version of Teleport, as well as images which are
+automatically rebuilt every night. These nightly images point to the latest version of Teleport from the current release branch, plus the two previous release branches.
+They are stable, and we recommend their use to easily keep your Teleport installation up to date.
+
+| Image name | Community or Enterprise? | Teleport version | Image automatically updated? | Image base |
+|---|---|---|---|---|
+| `quay.io/gravitational/teleport:4.3` | Community | The latest version of Teleport Community 4.3 | Yes | [Ubuntu 20.04](https://hub.docker.com/_/ubuntu) |
+| `quay.io/gravitational/teleport:4.3.0` | Community | 4.3.0 | No | [Ubuntu 18.04](https://hub.docker.com/_/ubuntu) |
+
+For testing, we always recommend that you use the latest release version of Teleport, which is currently `{{teleport.latest_oss_docker_image}}`.
+
+### Quickstart using docker-compose
+
+The easiest way to start Teleport quickly is to use `docker-compose` with our [`teleport-quickstart.yml`](https://github.com/gravitational/teleport/blob/master/docker/teleport-quickstart.yml) file:
```

Comment:
To be extra explicit, we should note that the user needs to install Docker on their machine.